src/speculators/models/eagle3/model_definitions.py

```
@@ -133,6 +133,358 @@ def __init__(
+#            * _patch_vwn_eagle3_projections 接收 norm_class 参数，同时适配 Llama/Qwen3
+#            * forward 按照 pre_vwn → A2 → Attn → B1 → A3 → FFN → B2 → A4 的顺序执行
+#   [新增] LlamaDecoderVwnEagle3FirstLayer — 对应 LlamaDecoderEagle3FirstLayer
+#   [新增] QwenDecoderVwnEagle3FirstLayer  — 对应 Qwen3DecoderEagle3FirstLayer
+#   [修改] model_classes 补充 llama，与 eagle3/model_definitions.py 保持一致
+
+
+# ──────────────────────────────────────────────────────────────────────────────
+# pre_vwn_layer_v0（主流程）
+# ──────────────────────────────────────────────────────────────────────────────
+
+class VwnEagle3PreVwnLayer(nn.Module):
```

Ruff (0.15.15)

[error] Undefined name `nn`

(F821)